server/dataone_register.py

```diff
@@ -40,14 +37,33 @@ def unesc(value):
     return urllib.parse.unquote_plus(value)
 
 
-def query(q, fields=["identifier"], rows=1000, start=0):
-    """Query a DataONE Solr index."""
+def query(q,
+          url=DataONELocations.prod_cn.value,
+          fields=["identifier"],
+          rows=1000,
+          start=0,
+          test=False):
+    """
+    Query a DataONE Solr index.
+    :param q: The query
+    :param url: The URL to the coordinating node
```

Comment:
Might rename this to `base_url` to conform with other arg names since it's not the URL to the coordinating node but a URL to the d1 API base url of the CN